libs/Makefile.shared-python-libs

```diff
@@ -19,7 +19,8 @@ pre-venv:
 	fi
 
 venv: pre-venv
-	uv sync ${PIP_INSTALL_PARAMS}
+	uv lock --check
```

Comment:
Why first do a check if you're going to sync anyway?

Reply:
This is mainly there for GHA. If someone adds a new dependency but does not update the lock file accordingly, it will fail the GHA for this library.